City-of-Helsinki/helsinki-paatokset
# PP-200: Decisions AJAX loading

Testing:

- Check that you have the latest migration data (no change to migrations in this branch)
- View an issue page with attached decisions. Selecting decisions from dropdown / arrow buttons should load them via AJAX. 
- Verify that loading works with JS disabled, as well
